Update granularities.md (#12712)
apache/druid: docs/querying/granularities.md

Supported granularity strings are: `all`, `none`, `second`, `minute`, `fifteen_minute`, `thirty_minute`, `hour`, `day`, `week`, `month`, `quarter` and `year`.

* `all` buckets everything into a single bucket

> * `none` actually does bucket data - to the granularity of the internal index - which means millisecond granularity. `none` can be thought of as `millisecond`.  Using `none` in a [TimeseriesQuery](../querying/timeseriesquery.md) is currently not recommended (the system will try to generate 0 values for all milliseconds that didn’t exist, which is often a lot).

Reviewer: ```suggestion
* `none` buckets data to the granularity of the internal index which is millisecond granularity. You can think of `none` as equivalent to `millisecond`.  Using `none` in a [TimeseriesQuery](../querying/timeseriesquery.md) is currently not recommended because the system will try to generate 0 values for all milliseconds that didn’t exist, which can be a lot.
```

> * `none` actually does bucket data - to the granularity of the internal index - which means millisecond granularity. `none` can be thought of as `millisecond`.  Using `none` in a [TimeseriesQuery](../querying/timeseriesquery.md) is currently not recommended (the system will try to generate 0 values for all milliseconds that didn’t exist, which is often a lot).

Reviewer 2: ```suggestion
* `none` is a slight misnomer since it buckets data to millisecond granularity—the granularity of the internal index. You can think of `none` as equivalent to `millisecond`. Do not use `none` in a [timeseries query](../querying/timeseriesquery.md). Druid fills empty interior time buckets with zeroes, meaning the output will contain results for every single millisecond in the requested interval.
```
